nix/overlay.nix

```diff
@@ -19,34 +19,16 @@ in {
           substituteInPlace src/rely/TestSuiteRunner.re --replace "Pervasives" "Stdlib"
         '';
       });
-      dream = osuper.dream.overrideAttrs (o: { doCheck = false; });
       alcotest = osuper.alcotest.overrideAttrs (o: {
         propagatedBuildInputs =
           prev.lib.lists.remove osuper.uuidm o.propagatedBuildInputs;
       });
-      qcheck = osuper.qcheck.overrideAttrs (_: {
-        src = prev.fetchurl {
-          url = "https://github.com/c-cube/qcheck/archive/v0.18.1.tar.gz";
-          sha256 = "1jzhwrzsf5290rs7hsa1my5yh1x95sh2sz92c4svd8yahzdlny7m";
-        };
-      });
```

Comment:
Moving a question here since this is the source of the diff.

From @aguillon 
> I don't usually follow what's going on with Nix scripts but I'm not sure why this was introduced in the first place, and why it has to go now? I believe qcheck is used in Pollinate and @gsebil08 and @rosalogia will probably use it more.

We needed a specific version of `qcheck` before this because they had a dependency on older `cmdliner`. `qcheck` is still available for use however :)